### PR TITLE
[en] Update admission-controllers.md

### PR DESCRIPTION
**Pull request by ptux**
- To make usage of requests and objects clear

### CODE REVIEW

File: content/en/docs/reference/access-authn-authz/admission-controllers.md [hunk 1]
```diff
@@ -30,9 +30,9 @@ mutating and validating (respectively)
 which are configured in the API.
 
 Admission controllers may be "validating", "mutating", or both. Mutating
-controllers may modify the objects they admit; validating controllers may not.
+controllers may modify the requests they admit; validating controllers may not.
```
Comment (divya-mohan0209) [inline content/en/docs/reference/access-authn-authz/admission-controllers.md:33]: They don't modify the requests, to the best of my knowledge. How about rephrasing the line as:

> Mutating
controllers may modify the request objects they admit; validating controllers may not.

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Rephrasing the sentence correctly since the requests themselves are not modified.
Lgtm otherwise.

File: content/en/docs/reference/access-authn-authz/admission-controllers.md (hunk 1 above)
Comment (ptux) [reply content/en/docs/reference/access-authn-authz/admission-controllers.md]: I got your point.
But `request objects` also confuses me.

**Comment by jimangel** [comment]
@ptux thanks for the updates! This looks ready to merge, would you mind please [squashing you commits](https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits)? Thanks!

/lgtm

**Comment by jimangel** [comment]
/lgtm
/approve

Thanks!

**Pull request merged**